vendor/github.com/openshift/cluster-api/pkg/apis/machine/v1beta1/machine_types.go

```diff
@@ -39,12 +39,6 @@ const (
 // Machine is the Schema for the machines API
 // +k8s:openapi-gen=true
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:name="Instance",type="string",JSONPath=".status.providerStatus.instanceId",description="Instance ID of machine created in AWS"
```

Comment:
Please, do not remove these lines. There are needed so we can generate CRDs with https://github.com/openshift/machine-api-operator/blob/master/install/0000_30_machine-api-operator_02_machine.crd.yaml#L9-L33